crates/sui-graphql-e2e-tests/tests/call/checkpoint_connection_pagination.move

```diff
@@ -6,99 +6,157 @@
 // Currently implemented only for items ordered in ascending order by `sequenceNumber`.
 
 // Assuming checkpoints 0 through 12
-// first: 4, after: "6" -> checkpoints 7, 8, 9, 10
-// first: 4, after: "6", before: "8" -> checkpoints 7
-// first: 4, before: "6" -> error
-// last: 4, after: "6" -> error
-// last: 4, before: "6" -> checkpoints 2, 3, 4, 5
-// last: 4, before: "6", after: "3" -> error
-// no first or last -> checkpoints 0, 1, 2, 3
+// first: 4, after: 6 -> checkpoints 7, 8, 9, 10
+// first: 4, after: 6, before: 8 -> checkpoints 7
+// first: 4, before: 6 -> checkpoints 0, 1, 2, 3
+// last: 4, after: 6 -> checkpoints 9, 10, 11, 12
+// last: 4, before: 6 -> checkpoints 2, 3, 4, 5
+// last: 4, after: 3, before: 6 -> checkpoints 4, 5
+// no first or last -> checkpoints 0, 1, ..., 11, 12
 // first: 4 -> checkpoints 0, 1, 2, 3
 // last: 4 -> checkpoints 9, 10, 11, 12
+// first: 4, last: 2 -> error
 
 //# init --addresses Test=0x0 --simulator
 
 //# create-checkpoint 12
 
-//# run-graphql
+//# run-graphql --cursors 6
```

Comment:
Does this initialize a number of cursors, or something else?

Reply:
It'll just baes64 encode the provided value https://github.com/MystenLabs/sui/blob/main/crates/sui-transactional-test-runner/src/test_adapter.rs#L1122

Reply:
Yeah this particular invocation reads confusingly because the cursor is a number and there's only one of them, but the flag is called `--cursors` because it allows you to serialize multiple values into opaque cursors, like `--cursors x y z`, and as Will mentioned, it does that by base64 encoding them, and binding them to variables for the E2E test interpolator, so we can refer to them as `@{cursor_0}`, `@{cursor_1}` etc.